### PR TITLE
LPD-41537 Allow Basic Get and Post operations of Children of Page Template Sets

### PR DESCRIPTION
**Pull request by lfbesada**
https://liferay.atlassian.net/browse/LPD-41537

### CODE REVIEW

**Comment by liferay-page-management** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

File: modules/apps/headless/headless-admin-site/headless-admin-site-impl/src/main/java/com/liferay/headless/admin/site/internal/resource/v1_0/PageTemplateResourceImpl.java [hunk 1]
```diff
@@ -90,6 +91,39 @@ public PageTemplate getSiteSiteByExternalReferenceCodePageTemplate(
 						siteExternalReferenceCode)));
 	}
 
+	@Override
+	public Page<PageTemplate>
+			getSiteSiteByExternalReferenceCodePageTemplateSetPageTemplatesPage(
+				String siteExternalReferenceCode,
+				String pageTemplateSetExternalReferenceCode, Boolean flatten)
+		throws Exception {
+
```
Comment (ealonso) [inline modules/apps/headless/headless-admin-site/headless-admin-site-impl/src/main/java/com/liferay/headless/admin/site/internal/resource/v1_0/PageTemplateResourceImpl.java:100]: @lfbesada I think you forgot the FeatureFlag check

**Comment by lfbesada** [comment]
@ealonso Thanks a lot for catching it :heart:

**Comment by ealonso** [comment]
ci:forward

**Pull request closed without merging**